### DISCUSSION TITLE
How to handle reactions

### DISCUSSION THREAD

**Discussion by ivan-buriak** [nutgram/nutgram#704] [Examples]
Could you please provide any example how to use onMessageReaction handler, cannot find anything about it in Docs
Many Thanx!

**Comment by Lukasss93** [comment]
```php
$bot = new Nutgram('xxxx');

$bot->onMessageReaction(function(Nutgram $bot){
    $reaction = $bot->messageReaction();
    $bot->sendMessage('You reacted with: '.$reaction->new_reaction[0]->emoji);
});

$bot->run();
``` 
![immagine](https://github.com/nutgram/nutgram/assets/4071613/0d6b6de3-da07-4676-8506-48d315eb9072)
